chemprop/conf.py

```diff
@@ -1,6 +1,7 @@
 """Global configuration variables for chemprop"""
-from chemprop.featurizers.molgraph.molecule import SimpleMoleculeMolGraphFeaturizer
 
+from chemprop.featurizers import SimpleMoleculeMolGraphFeaturizer, MultiHotAtomFeaturizer
 
-DEFAULT_ATOM_FDIM, DEFAULT_BOND_FDIM = SimpleMoleculeMolGraphFeaturizer().shape
+
+DEFAULT_ATOM_FDIM, DEFAULT_BOND_FDIM = SimpleMoleculeMolGraphFeaturizer(atom_featurizer=MultiHotAtomFeaturizer.default()).shape
```

Comment:
```suggestion
DEFAULT_ATOM_FDIM, DEFAULT_BOND_FDIM = SimpleMoleculeMolGraphFeaturizer().shape
```
we want the global configuration to abstractly depend on the default initialization of a `SimpleMoleculeMolGraphFeaturizer`, so we don't to specify what `atom_featurizer` we supply it. Specifying this makes it more possible that we what we want to be the default data transform and what our model assumes is the default can be decoupled.